.github/workflows/ci.yml

```
@@ -15,8 +15,21 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      # Set up the specific Python version for Black
+      - name: Set up Python 3.12
+        uses: actions/setup-python@v2
```

actionlint

the runner of "actions/setup-python@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)